F/Flowstar/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Flow*: A verification tool for cyber-physical systems
+# see: https://flowstar.org/examples/
+
+# LICENSE from https://flowstar.org/dowloads/ stating:
+# "The source code is released under the [GNU General Public License (GPL)](https://www.gnu.org/licenses/gpl-3.0.html). We are happy to release the code under a license that is more (or less) permissive upon request."
+
+using BinaryBuilder
+
+platforms = supported_platforms(exclude=Sys.iswindows)
+platforms = expand_cxxstring_abis(platforms)
+
+name = "Flowstar"
+version = v"2.1.0"
+sources = [
+    ArchiveSource("https://www.cs.colorado.edu/~xich8622/src/flowstar-$version.tar.gz", "642b17a55c6725d4bfc5b98900802e3b82f37fbbe9fb9028f1110c669a5afc86")
+    FileSource("https://www.gnu.org/licenses/gpl-3.0.txt", "3972dc9744f6499f0f9b2dbf76696f2ae7ad8af9b23dde66d6af86c9dfb36986"; filename = "LICENSE")
```

Comment:
We already ship the GPL license in the build environment, no need to download it
```suggestion
```

Reply:
If I don't include I LICENSE file I don't pass the audit. Do I need to copy it from somewhere?

Reply:
Oh... I see it in the script change below.

Reply:
That's what the line `install_license /usr/share/licenses/GPL-3.0+` I suggested adding in https://github.com/JuliaPackaging/Yggdrasil/pull/5035#discussion_r900024955 is for